### ISSUE TITLE
organizations_organizational_unit requires apply on every run for accounts argument when empty

### ISSUE THREAD

**Issue by brdly-smith** [terraform-providers/terraform-provider-aws#8733]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
Terraform v0.12.0-rc1
AWS Provider Versions:
v2.11.0
v2.10.0

### Affected Resource(s)

* aws_organizations_organizational_unit

### Terraform Configuration Files

```hcl
resource "aws_organizations_organizational_unit" "ou_testing" {
  name      = "OU Testing"
  parent_id = "${aws_organizations_organization.example.roots.0.id}"
}
```

### Expected Behavior
Should not detect any changes.

After applying succeeds, the next apply should not find any changes.

Accounts should be provided from parent_id relationships.

### Actual Behavior

Applies always detect changes.

```
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

   aws_organizations_organizational_unit.ou_testing will be updated in-place
  ~ resource "aws_organizations_organizational_unit" "ou_testing" {
        _+ accounts  = (known after apply)_
        arn       = "arn:aws:organizations::*:ou/o-*/ou-*"
        id        = "ou-*"
        name      = "OU Testing"
        parent_id = "r-*"
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```
### Steps to Reproduce

1. `terraform apply`

### References

* #0000

**Comment by bflad** [comment]
Hi @brdly-smith 👋 Sorry you are running into trouble there. From the output, this appears to be a prerelease of Terraform 0.12. Can you let us know which one (e.g. `terraform version`)? Thanks.

**Comment by brdly-smith** [comment]
Terraform v0.12.0-rc1

amended original

**Comment by ryndaniels** [comment]
Hi @brdly-smith 👋 thanks for clarifying the version! We believe that this has been fixed by the release of Terraform 0.12 - I'm unable to replicate this bug with 0.12.0 or later using this test config:

```
resource "aws_organizations_organization" "example" {

}

resource "aws_organizations_organizational_unit" "ou_testing" {
  name      = "OU Testing"
  parent_id = "${aws_organizations_organization.example.roots.0.id}"
}
```

Could you give this a try with the most recent Terraform version and let us know if you're still running into this issue? Thanks!

**Comment by ryndaniels** [comment]
Going ahead and closing this since we haven't gotten a response, but please feel free to re-open if you end up still running into this issue with the latest Terraform and AWS Provider versions! 🙂
